### DISCUSSION TITLE
[Art competition] Galois Connection

### DISCUSSION THREAD

**Discussion by YoungseokCh** [prosyslab-classroom/cs524-program-analysis#267] [Art competition]
![DALL·E 2022-10-31 11 45 22 - a robot in a starry night watching himself in the water surface, in the style of vaporwave](https://user-images.githubusercontent.com/37728816/198923130-38c837a4-a4f4-46ec-9906-489340a65884.png)
## Short Description
For me, [*Galois Connection*] means [*a robot in a starry night watching himself in the water surface, in the style of vaporwave*]".

## Long Description (up to 3 sentences)

A water surface reflects the original image in the real world with some resolution loss and little shape transformation.  The reflected image may different from the original image, however the entire shape is not collapsed so we can guess what the original image was. It shows some properties of Galois connection that abstraction may lose some data, but it should preserve the order (aka. monotonicity)!

**Comment by KihongHeo** [comment]
Hey Youngseok. This is an amazingly beautiful picture of Galois connection!
